### PR TITLE
Add part 2 of transforming data to graph blog post

### PR DESCRIPTION
**Pull request by prrao87**
This PR adds part two of the "Transforming your data to a graph" series. It focuses on the data science and analysis aspects.

- Analyzing disputed transactions
- Visualizing interesting subgraphs
- Running graph algorithms via NetworkX
- Interoperability with Pandas

### CODE REVIEW

File: blog/2024-02-23-transforming-your-data-2/index.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+52
+9
+```
+
+The first cluster of clients and merchants contains 52 nodes, while the second cluster contains 9
+nodes. This helps narrow down on a transaction subgraph that has a greater degree of connectivity.
+
+### Closeness centrality
+
+Centrality algorithms are among the most commonly used algorithms in graph data science. They are
```
Comment (semihsalihoglu-uw) [inline blog/2024-02-23-transforming-your-data-2/index.md:326]: I would give a high level motivation here. For example: "Centrality algorithms are among the most commonly used algorithms in graph data science. At a high-level, centrality algorithms are used to identify "important" nodes. Specifically nodes that serve the role of connecting many other nodes in the graph. Suppose the owner of our transaction graph is an e-commerce company. Suppose further that the company wants to promote its "important merchants" to attract more customers to its network. Centrality metrics, similar to the popular PageRank metric, are concrete ways to order nodes in terms of "importance". 

I would also change the "Closeness centrality[^2] is different from common algorithms like PageRank or degree centrality in that they indicate how _close_ a node is to all other nodes." sentence to just say "Closeness centrality[^2] measures how _close_ a node n is to all other nodes by calculating the average of the shortest path length from n to every other node in the graph."

File: blog/2024-02-23-transforming-your-data-2/index.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+used to detect nodes that spread information efficiently, or that are important in some way by
+measuring how central a node is relative to the group. Closeness centrality[^2] is different from
+common algorithms like PageRank or degree centrality in that they indicate how _close_ a node is
+to all other nodes. It is calculated as the average of the shortest path length from the node to
+every other node in the network in the graph.
```
Comment (semihsalihoglu-uw) [inline blog/2024-02-23-transforming-your-data-2/index.md:331]: typo. remove "in the network".

**Review by semihsalihoglu-uw — APPROVED** [review]

File: blog/2024-02-23-transforming-your-data-2/index.md [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+numTransactions |
+--- |
+1100 |
+
+The dataset contains 1,000 clients who made 1,100 transactions with 100 merchants. We can visualize
+the transactions with a particular merchant at a particular city and belonging to a particular
+company using the following query:
+
+```sql
+MATCH (c:Client)-[t:TransactedWith]->(m:Merchant)-[b:BelongsTo]->(co:Company)
```
[the remark this answers is not in the document]
Comment (prrao87) [reply blog/2024-02-23-transforming-your-data-2/index.md:145]: I wasn't aware we could just chain together multiple MATCH statements this way (somehow when I used Neo4j the habit of using WITH came in). Will keep this in mind, nice to know!

File: blog/2024-02-23-transforming-your-data-2/index.md [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+When running this query in Kùzu Explorer, we can customize the edge properties displayed in the graph visualization. In
+the following image, we mark the `TransactedWith` edges with the boolean value of the `is_disputed`
+property from the data. Only a small fraction of these transactions have the `is_disputed` property
+marked as `true`.
+
+<div class="img-center">
+<img src={QueryDisputedVicinity}/>
+</div>
+
+It can be seen that certain clients interacted with multiple merchants, some of which share common paths with
```
[the remark this answers is not in the document]
Comment (prrao87) [reply blog/2024-02-23-transforming-your-data-2/index.md:184]: It's not possible to change the color of only specific edges based on the property value in Kùzu explorer, but I've updated the image to circle the disputed transaction edges for now.

**Pull request merged**